runtime/src/blueoil_opencv.cpp

```diff
@@ -20,6 +20,7 @@ limitations under the License.
 #include "blueoil_opencv.hpp"
 
 namespace blueoil {
+
```

Comment:
it looks better to remove this blank line from the point of view of consistency.

Reply:
Probably, We shold be revise runtime/README.md slightly to import the results of the dll build.

Reply:
I added the DLK library import chapter to runtime/README.md . I'm poor english writer. Perhaps, we need to re-write the runtime/README.md .

Reply:
https://github.com/blue-oil/blueoil/pull/577/commits/54f68cd5894fd12b80a5053405780e0c004c79c5
https://github.com/blue-oil/blueoil/pull/577/commits/15130c121ba74f8da168cd4c12df4a6851e0062b